github3/pulls.py

```diff
@@ -208,6 +209,16 @@ def close(self):
         return self.update(self.title, self.body, 'closed')
 
     @requires_auth
+    def create_comment(self, body):
+        """Create a comment on this pull request's issue.
+
+        :param str body: (required), comment body
+        :returns: :class:`IssueComment <github3.issues.comment.IssueComment>`
+        """
+        issue = self.issue()
```

Comment:
Also, you can just use `self.comments_url` instead of `self.links['comments']['href']`.